bigquery_etl/stripe/event.schema.json

```diff
@@ -2132,6 +2132,141 @@
           }
         ]
       },
+      {
+        "name": "discount",
+        "type": "RECORD",
+        "fields": [
```

Comment:
copied from where it was already defined within `invoice` to it's own resource because `invoice.discount` is [deprecated](https://stripe.com/docs/api/invoices/object#invoice_object-discount) in favor of a list of ids at `invoice.discounts`.